figures/README.md

```diff
@@ -41,3 +42,121 @@ However, we list information about the resources, intermediate steps, and [PBTA
 | Figure | Individual script | Notes on requirements | Linked analysis modules | PBTA data files consumed |
 |--------|--------|------------------|-------------------------|-----------------------------|
 | Figure 2 | [scripts/fig2-mutational-landscape.R`](./scripts/fig2-mutational-landscape.R) | ~128MB of RAM are needed due to the run_caller_consensus_analysis-pbta.sh handling of large MAF files|[`snv-callers`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/snv-callers) <br> [`mutational-signatures`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/mutational-signatures) |  `pbta-snv-lancet.vep.maf.gz` <br> `pbta-snv-mutect2.vep.maf.gz` <br> `pbta-snv-strelka2.vep.maf.gz` <br> `pbta-snv-vardict.vep.maf.gz` <br> `tcga-snv-lancet.vep.maf.gz` <br> `tcga-snv-mutect2.vep.maf.gz` <br> `tcga-snv-strelka2.vep.maf.gz` |
+
+## Color Palette Usage
+
+This project has a set of unified color palettes.
+There are six sets of hex color keys to be used for all final figures.
+They are created by running `scripts/color-palettes.R` which creates 6 TSV files
+in the `figures/palettes` folder.
+Each palette TSV file has two columns of data: `hex_codes` and `color_names`.
+`hex_codes` contains the colors to be passed to your plotting code and `color_names` contains short descriptors of each color (e.g. `gradient_1`, or `divergent_neutral`).
+Each palette contains an `na_color` that is the same color in all palettes it
+is always the last value in the list.
+So depending on the circumstance, if `na_color` is not needed or will be supplied
+to a plot's argument in a different way, you can use a `dplyr::filter(hex_code != "na_color")`
+to remove the `na_color`.
+Biospecimens without a `short_histology` designation are assigned the `NA` color
+for this project.
```

Comment:
Some of these appear to be wrapped instead of one sentence per line.

Reply:
Habit. Whoops.